main.js

```diff
@@ -0,0 +1,112 @@
+var weather = {};
+var fiveDayForecast = [
+  [],
+  [],
+  [],
+  [],
+  []
+];
+
+
+$('.search').on('click', function () {
+  var search = $('#search-query').val();
+  fetch(search);
+});
+
+
+var addCurrentWeather = function(data){
+  var impTemp = Math.round((data.main.temp - 273.15) * 9/5 + 32);
```

Comment:
Read the docs to see that you could have asked for `imperial` temps: https://openweathermap.org/current